ci/run_autoscaling_osp18.yml

```diff
@@ -1,14 +1,24 @@
 ---
-- name: Telemetry tests on osp18 with autoscaling enabled
+- name: Run telemetry autoscaling tests on osp18
   hosts:  "{{ cifmw_target_hook_host | default('localhost')  }}"
   gather_facts: true
   environment:
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
+  vars_files:
+    - vars/common.yml
   tasks:
+    - name: Include setup tasks 
```

Comment:
Since this is for running autoscaling only, you can add the setup tasks directly instead of having a separate file for setup_env.

Reply:
we don't need the patch openstackversion with any other jobs ? logging